instrumentation/events/lttng-module/net.h

```diff
@@ -553,6 +553,175 @@ LTTNG_TRACEPOINT_EVENT_INSTANCE_MAP(net_dev_template,
 
 	TP_ARGS(skb)
 )
+
+#if (LINUX_VERSION_CODE >= KERNEL_VERSION(3,14,0))
+
+/* Trace events for the receive entry points */
+LTTNG_TRACEPOINT_EVENT_CLASS(net_dev_receive_entry_template,
+
+	TP_PROTO(const struct sk_buff *skb),
+
+	TP_ARGS(skb),
+
+	TP_FIELDS(
+		ctf_integer_hex(const void *, skbaddr, skb)
+	)
+)
+
+LTTNG_TRACEPOINT_EVENT_INSTANCE_MAP(net_dev_receive_entry_template,
+
+	napi_gro_frags_entry,
+
+	net_napi_gro_frags_entry,
```

Comment:
In the linux kernel, those tracepoints are in the net.h file. There is a napi.h file which only contains the napi_poll event, like in lttng. But here, namespacing the name of the event lik net_napi_gro... name does sound a bit out of place... Your call!

Reply:
lttng-modules has a strict requirement that the prefix is the name of the provider (TRACE_SYSTEM), here "net".

It would be weird to start mixing up net.h and napi.h into different provider implementations, so I think keeping the net_ prefix is fine, considering that the kernel tracepoint is within net.h.

Reply:
Excellent, so nothing to change here